### PR TITLE
Feat misc test additions

### PR DESCRIPTION
**Pull request by SanketD92**
Issue #, if available:
GALI-724

Description of changes:
Adding integration tests for User Roles, IP, External CFN template, and Costs APIs.

Checklist:

- [x] Have you successfully deployed to an AWS account with your changes?
- [x] Have you written new tests for your core changes, as applicable?
- [x] Have you successfully tested with your changes locally?

AS review ticket id:

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

**Review by ealemayehu — APPROVED** [review]
LGTM!

**Review by ealemayehu — DISMISSED** [review]

File: main/integration-tests/__test__/api-tests/costs/get-costs.test.js [hunk 1]
```diff
@@ -0,0 +1,72 @@
...
+    });
+
+    it('should pass if user is active', async () => {
+      const researcherSession = await setup.createResearcherSession();
+
+      const response = await researcherSession.resources.costs.getIndexCosts();
+
+      expect(response[0]).toHaveProperty('cost');
+      expect(response[0]).toHaveProperty('startDate');
+    });
```
Comment (jn1119) [inline main/integration-tests/__test__/api-tests/costs/get-costs.test.js]: Does this require some setup to pass? Is the assumption that our pipeline account has done this manual setup?

File: main/integration-tests/__test__/api-tests/template/get-external-cfn-template.test.js [hunk 2]
```diff
@@ -0,0 +1,66 @@
...
+  beforeAll(async () => {
+    setup = await runSetup();
+    adminSession = await setup.defaultAdminSession();
+  });
+
+  afterAll(async () => {
+    await setup.cleanup();
+  });
+
+  describe('Getting an external CFN template', () => {
```
Comment (jn1119) [inline main/integration-tests/__test__/api-tests/template/get-external-cfn-template.test.js:32]: Do we need to add tests for guests as well?

File: main/integration-tests/__test__/api-tests/costs/get-costs.test.js (hunk 1 above)
Comment (SanketD92) [reply main/integration-tests/__test__/api-tests/costs/get-costs.test.js]: Yes, need to enable cost explorer and activate the 3 tags. (I've enabled the cost explorer in the e2e env today.)

**Review by jn1119 — APPROVED** [review]

**Pull request merged**